packages/apollo/src/apollo.ts

```diff
@@ -33,25 +33,8 @@ export class ApolloPlugin<
   }
 
   public getApolloMiddleware() {
-    return async (resolve: any, root: any, args: any, context: any, info: any): Promise<any> => {
-      this.logger.runTask(() => {
-        let queryOrMutationName = 'unknown';
-        const operationNames = info.operation.selectionSet.selections.map(
-          (item: any) => item?.name?.value,
-        );
-
-        if (operationNames.length === 1) {
-          queryOrMutationName = `${operationNames[0]}`;
-        } else if (operationNames.length > 1) {
-          queryOrMutationName = JSON.stringify(operationNames);
-        }
-
-        this.logger.setContextProperty('queryName', queryOrMutationName).add({
-          level: this.options.level,
-          message: 'Graphql request initiated',
-          data: args,
-        });
-      });
+    return async (resolve: any): Promise<any> => {
+      return this.logger.runTask(resolve, true);
     };
```

Comment:
tady snad jen na vysvětlenou: podle mě tu není potřeba nic logovat, protože to samý se loguje hned vzápětí v `didResolveOperation()` - jen jsem tam přesunul to nastavení `queryName` do contextu - takhle ho aspoň za nás řeší Apollo a nemusíme se spoléhat na polointerní implementační detaily